packages/react-components/src/components/VideoGallery.tsx

```diff
@@ -350,7 +391,9 @@ export const VideoGallery = (props: VideoGalleryProps): JSX.Element => {
           /* @conditional-compile-remove(PSTN-calls) */
           participantState={participant.state}
           /* @conditional-compile-remove(pinned-participants) */
-          showRemoteVideoTileContextualMenu={props.showRemoteVideoTileContextualMenu}
+          showRemoteVideoTileContextualMenu={
+            remoteVideoTileMenuOptions && remoteVideoTileMenuOptions.kind === 'contextual'
```

Comment:
do we need a check here for `remoteVideoTileMenuOptions` as it will be always available as default options

Reply:
I had the same thought but yes because remoteVideoTileMenuOptions is a union type `false | VideoTileContextualMenuProps | VideoTileDrawerMenuProps` and could be `false`.